packages/lesswrong/components/posts/PostsNewForm.tsx

```diff
@@ -239,12 +247,14 @@ const PostsNewForm = () => {
     activateRSVPs: true,
     onlineEvent: groupData?.isOnline,
     globalEvent: groupData?.isOnline,
+    title: (query && query.title) ?? "Untitled Draft",
     types: query ? ['SSC', 'IFANYONE', 'PETROV'].filter(type => query[type]) : [],
     meta: query && !!query.meta,
     groupId: query && query.groupId,
     moderationStyle: currentUser && currentUser.moderationStyle,
     generateDraftJargon: currentUser?.generateJargonForDrafts,
-    postCategory
+    postCategory,
+    ...(query?.contents ? { contents: { originalContents: { type: "ckEditorMarkup", data: sanitize(query.contents) } } } : {}),
```

Comment:
I'm sanitizing this, but, if this is hella scary and we should be doing a different thing, seems understandable.